src/cljs/rems/administration/status_flags.cljs

```diff
@@ -0,0 +1,50 @@
+(ns rems.administration.status-flags
+  (:require [re-frame.core :as rf]
+            [rems.text :refer [text]]))
+
+(defn- disable-button [update-item-cmd item]
+  [:button.btn.btn-secondary.button-min-width
+   {:type "button"
+    :on-click #(rf/dispatch [update-item-cmd (assoc item :enabled false)])}
```

Comment:
I think I'd prefer to have full blown callback functions here so and not only passing effect names.

Reply:
Would it be better for the callback to receive as parameter the updated item or just the enabled boolean?

Reply:
The `(if (:enabled item)` already makes `enabled-toggle` aware of the existence of `:enabled`, so I think I'll make the callback receive the updated item. Otherwise for full decoupling it would be necessary to parameterize also the current value of `:enabled`.

Reply:
Done.